source/_integrations/acaia.markdown

```
@@ -0,0 +1,105 @@
+{% configuration_basic %}
+MAC:
+  description: "The MAC address of your scale."
+  required: true
+  type: string
+Is new style scale:
+  description: "Check this if your scale is a newer model (manufactured after 2021). This setting affects how commands are sent to the scale. If your scale doesn't respond to commands, try toggling this setting. For example, Lunar models made after 2021 should have this enabled."
+  required: false
+  type: boolean
+{% endconfiguration_basic %}
+
+# Available platforms & entities
```

Markdownlint

Multiple top-level headings in the same document

(MD025, single-title, single-h1)